dbt/profiles.yml

```diff
@@ -3,11 +3,19 @@ dbt-civis:
   default: redshift_adapter
   outputs:
     redshift_adapter:
-      type: redshift
+      type: "{{ env_var('DATABASE_TYPE', 'redshift') }}"
       host: "{{ env_var('DATABASE_HOST') }}"
       port: "{{ env_var('DATABASE_PORT') | as_number }}"
       user: "{{ env_var('DATABASE_USERNAME') }}"
       pass: "{{ env_var('DATABASE_PASSWORD') }}"
       dbname: "{{ env_var('DATABASE_NAME') }}"
-      schema: "{{ env_var('DATABASE_SCHEMA', 'default_schema_name') }}"
+      schema: "{{ env_var('DATABASE_SCHEMA') }}"
```

Comment:
do we not want to allow defaulting like this?

Reply:
whether we include it in the example or not, people will still be allowed to pass in a default.

I created a readme for the dbt example and linked the dbt env var documentation on it. i think that's probably enough

Reply:
I think `DATABASE_SCHEMA` should be `DBT_SCHEMA`

Reply:
so are we going to change the backend?

Reply:
No I just mean that the correct name of the variable is `DBT_SCHEMA` and not `DATABASE_SCHEMA`. I think just changing it here will do the trick.

Reply:
No, I just mean that the environment variable is misnamed here; it should be `DBT_SCHEMA` not `DATABASE_SCHEMA`. If you look at the [profiles.yml](https://github.com/civisanalytics/docker-scratch/blob/team-onyx-dbt-jaffle_shop_duckdb-copy/profiles.yml#L29) docker scratch branch, it uses the correct environment variable name.

Reply:
oh i must have been looking at an old version of console. I see it is `dbt_schema` in the backend now!